pkg/tablebackup/basebackup.go

```diff
@@ -19,8 +19,13 @@ import (
 	"github.com/ikitiki/logical_backup/pkg/utils"
 )
 
+var TableNotFoundError = fmt.Errorf("table not found")
```

Comment:
there's no formatting at all. why not use `errors.New()` ?

Reply:
one less direct import, and fmt.Errorf ultimately does the same.

Reply:
Changed in the latest commit.